src/IdentityManager2/Assets/Scripts/App/ttIdmUsers.js

```diff
@@ -85,7 +85,7 @@
                 var props = properties.map(function (item) {
                     return {
                         type: item.meta.type,
-                        value: item.data
+                        value: item.data.toString()
```

Comment:
this is actually needed to allow serialization of values (int, string, what-not) into https://github.com/IdentityManager/IdentityManager2/blob/449d0676ce91aeec9a8c77fcc5853c48fca88847/src/IdentityManager2/Core/PropertyValue.cs#L10